pkg/cmd/gc/pods/pods.go

```diff
@@ -121,3 +157,28 @@ func (o *Options) MatchesPod(pod *corev1.Pod) (bool, time.Duration) {
 	}
 	return age > o.Age, age
 }
+
+// findJXNamespaces looks for namespace names that are owned by Jenkins X
+func (o *Options) findJXNamespaces(ctx context.Context) ([]string, error) {
+	var matched []string
+	namespaces, err := o.KubeClient.CoreV1().Namespaces().List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return []string{}, errors.Wrap(err, "cannot list namespaces to look for pods")
+	}
+	for _, n := range namespaces.Items { //nolint:gocritic
```

Comment:
why nolint? Does it give false positive? or is it because of `_, n` instead of using `for k := range ...`

Reply:
Do you have a suggestion? It says loop iteration = ~400 bytes and I should use a pointer I don't have an idea how to do this, Kubernetes librarry does not return a pointer.

Reply:
```
for k := range namespaces.Items {
  n := namespaces.Items[k]
  labels := n.GetLabels()
```

Reply:
Thanks, will try it :grin:

Reply:
Thanks, fixed.